app/core/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
+from django.urls import path
+
+URLPATTERNS = [
+    path('admin/', admin.site.urls),
```

Comment:
is this URL required? Can't think of a use case

Reply:
No, it can go. I will replace with a `ping` endpoint so there is something to hit.